core/primitives-core/src/config.rs

```diff
@@ -611,13 +611,10 @@ pub enum ExtCosts {
     alt_bn128_g1_sum_base,
     #[cfg(feature = "protocol_feature_alt_bn128")]
     alt_bn128_g1_sum_byte,
-
-    // NOTE: this should be the last element of the enum.
-    __count,
 }
 
 // Type of an action, used in fees logic.
-#[derive(Copy, Clone, Hash, PartialEq, Eq, Debug, PartialOrd, Ord)]
+#[derive(Copy, Clone, Hash, PartialEq, Eq, Debug, PartialOrd, Ord, EnumCount, Display)]
```

Comment:
I'd argue that “Display” is way too generic of a name to use in `derive` without referring to it by a crate (i.e. `strum::Display`), but this is probably a discussion to be had outside of this PR.

Reply:
Uhu, same thoughts here

Reply:
I'd even say this is not specific to `Display`: `strum` is a minor crate, with a short name, so we should just `strum::EnumCount, strum::Display`, etc.

Reply:
Yeah, I kinda prefer taking that approach to the extreme of “all external derives must be prefixed, and not imported", but either way this seems like a discussion best had over a changeset to the style guide.